JSDemos/Demos/List/ListSelection/description.md

```diff
@@ -1 +1,11 @@
-The **List** UI component offers three item selection modes: _&laquo;all&raquo;_, _&laquo;single&raquo;_ and _&laquo;multi&raquo;_. Note that when the current mode is _&laquo;all&raquo;_, the **List** allows a&nbsp;user to&nbsp;select all items at&nbsp;once by&nbsp;clicking the _Select All_ check box. The **selectAllMode** property allows you to&nbsp;decide whether all items should be&nbsp;selected across all pages or&nbsp;within the currently rendered pages only.
\ No newline at end of file
+The **List** offers four selection modes: *none*, *all*, *single*, and *multi*. 
+
+Selection is disabled by default. To enable it, pass one of the mentioned above values to the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property.
```

Comment:
- "none" isn't really a selection mode, since it disables item selection.
- Making the entire description about the demo is bad, but so is not referring to the demo at all.

```suggestion
The **List** offers three item selection modes: *"single"*, *"multiple"*, and *"all"*.  To enable item selection, pass a mode name to the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property. If you set this property to *"none"* (default setting), item selection is disabled. In this example, you can try out all the selection modes. Use the Selection Mode drop-down menu to switch between them.
```